shared/test/Datadog.Trace.ClrProfiler.Native.Tests/string_test.cpp

```diff
@@ -17,16 +17,18 @@ TEST(string, ToString)
 
     EXPECT_TRUE("42" == ToString(42));
 
+#ifndef LINUX
     EXPECT_TRUE("LPTSTR String" == ToString(TEXT("LPTSTR String")));
     EXPECT_TRUE("\tLPTSTR String\0" == ToString(TEXT("\tLPTSTR String\0")));
+#endif
```

Comment:
Should we test what we expect when it _is_ linux? Because what _should_ it be? 😅

Reply:
On Linux, there is no `TEXT` macro.  For string, this would add `L` before the string, which means "Unicode". But that's already what we have with `WStr`....
so I do not know what to do and what to expect on that.
From what I understood, it's specific to Windows, so be it :)